torch/csrc/jit/passes/quantization.cpp

```diff
@@ -692,6 +695,140 @@ c10::optional<IValue> toTwoElementIntList(Value* v) {
   }
   return c10::nullopt;
 }
+
+script::Module findChildModule(
+    const script::Module& module,
+    const std::vector<std::string>& path) {
+  script::Module m = module;
+  for (size_t i = 0; i < path.size(); ++i) {
+    m = m.attr(path[i]).toModule();
+  }
+  return m;
+}
+
+std::string addChildModule(
+    script::Module module,
+    const script::Module& child_module,
+    const std::vector<std::string>& path) {
+  script::Module cur = module;
+  for (size_t i = 0; i < path.size() - 1; ++i) {
```

Comment:
There should be a check for `path.size() == 0`, otherwise this loop will iterate until `UINT_MAX`.

Reply:
added filter for `path.size() == 0` earlier